src/flux/store.js

```diff
@@ -160,12 +194,21 @@ class Store extends EventEmitter {
       case Constants.LOG_OUT:
         this.logOut();
         break;
-      case Constants.LOAD_EXAMPLE:
-        this.loadExample(payload);
+      case Constants.LOAD_FLOW:
+        this.loadFlow(payload);
         break;
       case Constants.SHOW_POD_IN_FLOW:
         this.showPodByLabel(payload);
         break;
+      case Constants.CREATE_NEW_FLOW:
+        this.createNewFlow();
+        break;
+      case Constants.DUPLICATE_FLOW:
+        this.createNewFlow(payload);
```

Comment:
Two times the same function but one time you call it with an argument, it's not clear why without reading the function and reading until the middle. 
Why not `flowFromYAML(someYaml)` and different actions can do `flowFromYaml(DEFAULT_YAML)` or`flowFromYaml(selectedYAML)`